content/projects/gsoc.adoc

```diff
@@ -121,14 +123,19 @@ Organizational questions:
 * _jenkinsci-gsoc-all-public@googlegroups.com_ - sync-ups on organizational topics (meeting scheduling, process Q&A)
 * _jenkinsci-gsoc-orgs@googlegroups.com_ - for private communications with Jenkins GSoC Org Admins (escalations)
 
+[[officehours]]
```

Comment:
Redundant, as every header gets a link autogenerated. Or is the extra hypen so bad?